Endatix.sln

```diff
@@ -31,6 +31,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Endatix.Api.Host", "src\End
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Endatix.Persistence.SqlServer", "src\Endatix.Persistence.SqlServer\Endatix.Persistence.SqlServer.csproj", "{0047C6EE-7051-41CE-B72A-BECBFDA43341}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Endatix.Samples.SelfHosted", "samples\Endatix.Samples.SelfHosted\Endatix.Samples.SelfHosted.csproj", "{7C38EFD4-B5DB-4128-A180-849E54EB80F2}"
```

Comment:
@orilux  - you need to remove this reference too.

Reply:
Please ignore. Let's leave it as it is and all samples will remain referenced in the main solution for ease of use for fellow developers. This is how MediatR have also structured their repo